### PR TITLE
[Py] Added operators GroupConvolution and RNNCell to Python API.

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: doc/sphinx/source/python_api/_autosummary/ngraph.ops.rst [hunk 1]
```diff
@@ -76,6 +77,7 @@ ngraph.ops
       replace_slice
       reshape
       reverse
+      RNNCell
```
Comment (postrational) [inline doc/sphinx/source/python_api/_autosummary/ngraph.ops.rst]: Why are we not following the case convention here? We should have `rnn_cell` in this list, not `RNNCell`

File: python/ngraph/ops.py [hunk 2]
```diff
@@ -243,6 +329,7 @@ def mvn(data, axes, normalize_variance, eps, name=None):
     """
     return MVN(data, AxisSet(axes), normalize_variance, eps)
 
+<<<<<<< HEAD
```
Comment (postrational) [inline python/ngraph/ops.py]: Merge conflict artifact.

File: src/contrib/mlir/compiler/pass/mlir_subgraph_extraction.cpp [hunk 3]
```diff
@@ -456,6 +456,24 @@ bool MLIRSubgraphExtractionPass::is_supported_mlir_op(std::shared_ptr<Node> node
                std::all_of(window_dilation.begin(), window_dilation.end(), is_one);
     }
 
+    if (TI(ngraph::op::Convolution) == TI(*node))
```
Comment (postrational) [inline src/contrib/mlir/compiler/pass/mlir_subgraph_extraction.cpp]: Left over merge artifact.

File: python/ngraph/ops.py [hunk 4]
```diff
@@ -885,38 +1057,6 @@ def broadcast_to(node, new_shape, axis=None, name=None):
 
 
 @nameable_op
-def fake_quantize(data, input_low, input_high, output_low, output_high, levels, name=None):
```
Comment (postrational) [inline python/ngraph/ops.py]: I don't think you meant to remove `fake_quantize` with this PR.

Comment (Ewa21) [reply python/ngraph/ops.py]: Right. I restored it.

**Pull request merged**